aspnetcore/fundamentals/servers/kestrel/endpoints.md

```diff
@@ -85,6 +87,13 @@ Creates a configuration loader for setting up Kestrel that takes an <xref:Micros
 }
 ```
 
+If reloading configuration is enabled and a change is signaled then the following steps are taken:
+- The new configuraiton is compared to the old one, any endpoint without configuration changes are not modified.
+- Removed or modified endpoints are drained and stopped using a 5 second timeout.
```

Comment:
drained is slang that won't machine translate (MT). It's well known slang so maybe that's OK. Do you have an alternative?

Reply:
```suggestion
- Removed or modified endpoints are given 5 seconds to complete processing requests and shut down.
```